Api/KanbanizeApiV1.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Chaplean\Bundle\KanbanizeApiClientBundle\Api;
+
+use Chaplean\Bundle\ApiClientBundle\Api\AbstractApi;
+use Chaplean\Bundle\ApiClientBundle\Api\Parameter;
+use Chaplean\Bundle\ApiClientBundle\Api\Route;
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class KanbanizeApiV1.
+ *
+ * @method Route getProjectsAndBoards()   Get projects and boards     ( https://kanbanize.com/api/#get_projects_and_boards )
+ * @method Route postBoardStructure()     Get board structure         ( https://kanbanize.com/api/#get_board_structure )
+ * @method Route postFullBoardStructure() Get full board structure    ( https://kanbanize.com/api/#get_full_board_structure )
+ * @method Route postBoardSettings()      Get board settings          ( https://kanbanize.com/api/#get_board_settings )
```

Comment:
Are you sure that post makes get ??

Reply:
absolutely
`{
    "status": false,
    "error": "Unknown method. Please make sure you are sending an HTTP POST request."
}`